RocketChat/Rocket.Chat
# chore: Add `rid` to scope of `create-invite-links` permission check

## Proposed changes (including videos or screenshots)



This PR fixes a bug where users with the `owner` role can't create invites because the button isn't being displayed for him, to solve this issue the `rid` was added to the scope of the check.



## Issue(s)

Closes #27570

## Steps to test or reproduce

- Check that `owner` has privileges to create invites under admin permission settings
- Create a channel as a regular user
- Go to the member list
- Look for the Invite button next to Add button

### Expected Behavior

- The invite button exists and is visible

### Actual Behavior

- The invite button is missing

## Further comments

[TC-416]


[TC-416]: https://rocketchat.atlassian.net/browse/TC-416?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
